packages/apps/tools/src/pages/account/account-transactions/results/index.tsx

```diff
@@ -121,8 +133,71 @@ const CheckTransactions: FC = () => {
     await router.push(Routes.ACCOUNT_TRANSACTIONS_FILTERS);
   }
 
+  const handleOpenTransactionDetails = (result: ITransaction): void => {
+    setTransactionDetails(result);
+    // @ts-ignore
+    transactionDetailsRef.openSection(0);
+  };
+
   return (
-    <div>
+    <div className={mainContentClass}>
+      <DrawerToolbar
+        ref={transactionDetailsRef}
+        sections={[
+          {
+            icon: SystemIcon.Information,
+            title: t('Transaction Details'),
+            children: (
+              <>
+                <TrackerCard
+                  labelValues={[
+                    {
+                      label: 'Sender',
+                      value: transactionDetails?.fromAccount,
+                      isAccount: true,
+                    },
+                    {
+                      label: 'From chain',
+                      value: transactionDetails?.chain,
+                    },
+                  ]}
+                  icon={ProductIcon.QuickStart}
+                />
+                <Box marginBottom="$5" />
+                <ProgressBar
+                  checkpoints={[
+                    {
+                      title: 'Initiated transaction',
+                      status: 'complete',
+                    },
+                    {
+                      title: 'Transaction complete',
+                      status: 'complete',
+                    },
+                  ]}
+                />
+                <Box marginBottom="$2" />
+                <TrackerCard
+                  labelValues={[
+                    {
+                      label: 'Receiver',
+                      value: transactionDetails?.fromAccount,
+                      isAccount: true,
+                    },
+                    {
+                      label: 'to chain',
+                      value:
+                        transactionDetails?.crossChainId ||
+                        transactionDetails?.chain,
+                    },
+                  ]}
+                  icon={ProductIcon.ReceiverInactive}
```

Comment:
The icon should change according to the transfer status. If the transfer is complete then it should show the `Receiver` icon instead of `ReceiverInactive`

Reply:
Oh ok I see. Then maybe just replace the Icon with `ProductIcon.Receiver`